fix(register): adjust and improve broken styling for auth pages

nathan-power opened the pull request:
<!--  Thanks for sending a pull request!
If this is your first time, please read our contributor guidelines: https://github.com/Satellite-im/Core-PWA/wiki/Contributing
-->

**What this PR does** 📖
- Fixes styling for register related pages.
- Creates InfoModal.
- Moves the pin-info section in the signup unlock page into the InfoModal. 

**Which issue(s) this PR fixes** 🔨
AP-2022
<!--AP-X-->

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

WanderingHogan commented:
Lets try to get Nicholas' wallet PR in first if it's ready to go before merging this

WanderingHogan commented:
actually, there are no conflicting files with his PR, sorry, removing merge blocked tag

aewing commented on components/ui/InfoButton/InfoButton.vue:
```diff
@@ -0,0 +1,136 @@
+<template>
```
Nice. Very clean.

molimauro reviewed (changes requested):
Look good i just have a minor question :) Why have we added a bounce effect on the info modal?

nathan-power commented:
> Looks good i just have a minor question :) Why have we added a bounce effect on the info modal?

@molimauro 
Good call. I've replaced the internal animation styling of InfoModal by wrapping the UiModal component inside of it. We can keep the animation styling centralized this way as well.

nathan-power commented:
> Looks good i just have a minor question :) Why have we added a bounce effect on the info modal?

@molimauro
Good call. I've replaced the internal animation styling of InfoModal by wrapping the UiModal component inside of it. We can keep the animation styling centralized this way as well.

molimauro reviewed (approved):
Dope, thanks!
